designer/client/conditions/inline-conditions.js

```diff
@@ -59,15 +59,11 @@ class InlineConditions extends React.Component {
   }
 
   setState (state, callback) {
-    if (state.conditions) {
-      this.props.conditionsChange(state.conditions, undefined)
-    }
     super.setState(state, callback)
   }
 
```

Comment:
remove this setState definition if you're not doing any custom logic. this.setState will call the inherited (super.setState)

Reply:
Done